src/wp-signup.php

```diff
@@ -131,7 +131,7 @@ function show_blog_form( $blogname = '', $blog_title = '', $errors = '' ) {
 	$errmsg_blogname_aria = '';
 	if ( $errmsg_blogname ) {
 		$errmsg_blogname_aria = 'wp-signup-blogname-error ';
-		echo '<p class="error" id="wp-signup-blogname-error">' . $errmsg_blogname . '</p>';
+		echo '<p class="notice notice-error" id="wp-signup-blogname-error">' . $errmsg_blogname . '</p>';
```

Comment:
In `wp-signup.php`, these classes are defined in the `wpmu_signup_stylesheet` function added to `wp_head`. They aren't part of the WP Admin style group. While we could change the styles in the function, removing the `.error` class is likely to break custom styles implemented by sites. Should probably revert this change and go back to `.error` in this file.

Reply:
Resolved with https://github.com/WordPress/wordpress-develop/pull/7800/commits/8a1a8f47d33bbdd2f7f832ed2573c15f7433e244